internal/engine/upper.go

```diff
@@ -935,6 +935,7 @@ func handleDockerComposeLogAction(state *store.EngineState, action DockerCompose
 
 	dcState, _ := ms.ResourceState.(dockercompose.State)
 	ms.ResourceState = dcState.WithCurrentLog(model.AppendLog(dcState.CurrentLog, action, state.LogTimestamps))
+	state.Log = model.AppendLog(state.Log, action, state.LogTimestamps)
```

Comment:
1. This change is modifying the global tilt log, which is already working for this case, and this change would result in the message being appended twice.
2. This should instead be `ms.CombinedLog = model.AppendLog(ms.CombinedLog, action, state.LogTimestamps)`.